plugin.yml

```diff
@@ -3,7 +3,7 @@ name: PocketRadio
 main: xenialdan\PocketRadio\Loader
 version: 4.2.2
 api: [3.10.1]
-mcpe-protocol: [388,389,390]
+mcpe-protocol: 422
```

Comment:
I'd prefer to enable support for all protocol versions between as well

Reply:
I can add it back and re PR if you want. I was just getting it to work for 1.16.200